15-03-2024 10:24:28 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 10:24:32 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:33 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:33 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:33 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:36 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:36 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:37 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:37 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:38 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:38 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:38 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:38 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:39 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:39 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:40 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:40 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:40 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:40 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:40 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:40 [38;5;25m[INFO][0m django.server: - Broken pipe from ('127.0.0.1', 49390)
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:40 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:41 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:41 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:41 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:41 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:42 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:42 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:42 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:42 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:42 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:43 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:43 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:24:43 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:24:43 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:24:54 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 10:24:56 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:24:56 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:25:51 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\charl\.CompilerPrograms\Python\Lib\site-packages\django\contrib\admin\exceptions.py changed, reloading.
15-03-2024 10:26:10 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 10:26:13 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:26:13 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:19 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 10:28:22 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:22 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:27 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:27 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:27 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:27 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:27 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:27 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:27 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:27 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:28 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:28 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:28 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:28 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:28 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:28 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:28 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:29 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:29 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:29 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:29 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:28:29 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:28:29 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:28:29 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:28:29 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:31:27 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:31:27 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:31:30 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 302 0
15-03-2024 10:31:30 [38;5;25m[INFO][0m django.server: "GET /accounts/login/?next=/user/ HTTP/1.1" 200 4071
15-03-2024 10:31:30 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:31:31 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 302 0
15-03-2024 10:31:31 [38;5;25m[INFO][0m django.server: "GET /accounts/login/?next=/user/ HTTP/1.1" 200 4071
15-03-2024 10:31:31 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:31:35 [38;5;25m[INFO][0m django.server: "POST /accounts/login/?next=/user/ HTTP/1.1" 302 0
15-03-2024 10:31:37 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:31:37 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:31:48 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3712
15-03-2024 10:31:49 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:32:01 [38;5;25m[INFO][0m django.server: "GET /groups/search/ HTTP/1.1" 200 2967
15-03-2024 10:32:01 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:32:06 [38;5;25m[INFO][0m django.server: "GET /groups/search/?q=c HTTP/1.1" 200 4182
15-03-2024 10:32:06 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:32:09 [38;5;25m[INFO][0m django.server: "GET /groups/view/2/ HTTP/1.1" 200 2880
15-03-2024 10:32:09 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:32:13 [38;5;25m[INFO][0m django.server: "POST /groups/leave/2/ HTTP/1.1" 302 0
15-03-2024 10:32:14 [38;5;25m[INFO][0m django.server: "GET /groups/view/2/ HTTP/1.1" 200 2885
15-03-2024 10:32:14 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:32:18 [38;5;25m[INFO][0m django.server: "POST /groups/join/2/ HTTP/1.1" 302 0
15-03-2024 10:32:19 [38;5;25m[INFO][0m django.server: "GET /groups/view/2/ HTTP/1.1" 200 2880
15-03-2024 10:32:19 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:32:23 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 10:32:25 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:32:25 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:36:22 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 10:36:27 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 10:36:29 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:36:29 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:36:35 [38;5;25m[INFO][0m django.server: "GET /user/journeys/create HTTP/1.1" 200 9543
15-03-2024 10:36:35 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:36:53 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 10:36:55 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:36:55 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:37:01 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3712
15-03-2024 10:37:01 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:37:19 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 10:37:21 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:37:21 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:41:33 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 10:41:35 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:41:35 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:41:39 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:41:39 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:41:47 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 10:41:49 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:41:49 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:42:45 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:42:45 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:42:45 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 10:42:45 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 10:51:45 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 10:51:48 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
15-03-2024 10:51:48 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:51:55 [38;5;25m[INFO][0m django.server: "GET /login/ HTTP/1.1" 200 4071
15-03-2024 10:51:55 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:51:59 [38;5;25m[INFO][0m django.server: "POST /login/ HTTP/1.1" 302 0
15-03-2024 10:52:01 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:52:01 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:52:08 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3712
15-03-2024 10:52:08 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:55:23 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 10:55:27 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 10:55:29 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 10:55:29 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 10:55:34 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3712
15-03-2024 10:55:34 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 11:01:54 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 11:01:58 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 11:02:00 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 11:02:00 [38;5;220m[WARNING][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 404 1831
15-03-2024 11:04:46 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\charl\.GroupSoftwareDev\ECM2434_Group_Software_Engineering_Project\carbonCommuter\carbonCommuter\settings\base.py changed, reloading.
15-03-2024 11:04:47 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 11:04:51 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 11:04:53 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4309
15-03-2024 11:04:53 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8854
15-03-2024 11:04:53 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/forum.jpg HTTP/1.1" 200 627536
15-03-2024 11:05:00 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3712
15-03-2024 11:42:54 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 11:44:01 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 11:44:08 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 11:44:10 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
15-03-2024 11:44:10 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8854
15-03-2024 11:44:10 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/forum.jpg HTTP/1.1" 304 0
15-03-2024 11:44:11 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 11:44:11 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 11:44:29 [38;5;25m[INFO][0m django.server: "GET /user/profile/jskinner HTTP/1.1" 200 2978
15-03-2024 11:44:35 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
15-03-2024 11:44:35 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 11:44:49 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 4159
15-03-2024 11:45:38 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\common\utils.py changed, reloading.
15-03-2024 11:45:38 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 11:49:10 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 11:49:11 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 11:49:22 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 4159
15-03-2024 11:49:22 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 11:49:37 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 11:49:37 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 11:49:51 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 4159
15-03-2024 11:50:10 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6007
15-03-2024 11:51:19 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6007
15-03-2024 11:51:19 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 11:52:55 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6014
15-03-2024 11:52:55 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 11:53:36 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6014
15-03-2024 11:54:46 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6007
15-03-2024 11:54:46 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8856
15-03-2024 11:55:48 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6003
15-03-2024 11:55:48 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 11:56:30 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6003
15-03-2024 11:56:30 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8853
15-03-2024 11:57:57 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6021
15-03-2024 11:57:57 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 11:58:28 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6021
15-03-2024 11:58:28 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:05:14 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6978
15-03-2024 12:05:14 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:06:31 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7248
15-03-2024 12:06:31 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:08:17 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7248
15-03-2024 12:08:17 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8854
15-03-2024 12:08:41 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7248
15-03-2024 12:08:41 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8860
15-03-2024 12:10:24 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6978
15-03-2024 12:10:24 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8853
15-03-2024 12:13:05 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6493
15-03-2024 12:13:05 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:13:27 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6487
15-03-2024 12:13:27 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:13:48 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6492
15-03-2024 12:13:48 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:14:08 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6493
15-03-2024 12:14:36 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6497
15-03-2024 12:14:36 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:14:55 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6493
15-03-2024 12:19:58 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6493
15-03-2024 12:19:58 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:21:01 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:21:01 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:22:10 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:22:10 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8867
15-03-2024 12:22:37 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:22:37 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8871
15-03-2024 12:23:14 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:23:14 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8879
15-03-2024 12:23:39 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:23:39 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8875
15-03-2024 12:23:50 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:23:50 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:24:21 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:24:21 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8864
15-03-2024 12:24:39 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:24:39 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:24:46 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:24:46 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8871
15-03-2024 12:25:31 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:25:31 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8867
15-03-2024 12:25:46 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:25:46 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8875
15-03-2024 12:26:54 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:26:54 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8864
15-03-2024 12:28:17 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:28:18 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8865
15-03-2024 12:28:28 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:28:28 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8866
15-03-2024 12:28:48 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:28:48 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8873
15-03-2024 12:29:00 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:29:00 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8870
15-03-2024 12:29:11 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6526
15-03-2024 12:29:11 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8870
15-03-2024 12:29:31 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 63, in leaderboard
    journey_week_monday = journey.time_finished - timedelta(days=journey.time_finished.weekday())
AttributeError: 'NoneType' object has no attribute 'weekday'
15-03-2024 12:29:31 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 71481
15-03-2024 12:29:50 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 63, in leaderboard
    journey_week_monday = journey.time_finished - timedelta(days=journey.time_finished.weekday())
AttributeError: 'NoneType' object has no attribute 'weekday'
15-03-2024 12:29:50 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 71481
15-03-2024 12:30:08 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 63, in leaderboard
    journey_week_monday = journey.time_finished - timedelta(days=journey.time_finished.weekday())
AttributeError: 'NoneType' object has no attribute 'weekday'
15-03-2024 12:30:08 [38;5;25m[INFO][0m django.server: - Broken pipe from ('127.0.0.1', 61471)
15-03-2024 12:32:10 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7068
15-03-2024 12:32:10 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8866
15-03-2024 12:32:43 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 12:32:47 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 12:32:55 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7068
15-03-2024 12:32:55 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:34:23 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7068
15-03-2024 12:34:23 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:35:10 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6795
15-03-2024 12:35:10 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:36:31 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6795
15-03-2024 12:36:31 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9158
15-03-2024 12:37:02 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6795
15-03-2024 12:37:02 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9172
15-03-2024 12:37:23 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6795
15-03-2024 12:37:23 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9166
15-03-2024 12:37:29 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6795
15-03-2024 12:37:29 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:38:50 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6799
15-03-2024 12:38:50 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:39:52 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6799
15-03-2024 12:39:52 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9158
15-03-2024 12:40:04 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6797
15-03-2024 12:40:04 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:40:33 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6797
15-03-2024 12:40:34 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9156
15-03-2024 12:41:05 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 6797
15-03-2024 12:41:05 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9160
15-03-2024 12:47:45 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 12:47:47 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 12:47:56 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7067
15-03-2024 12:47:56 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 12:48:01 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7067
15-03-2024 12:48:10 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 12:48:11 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 12:48:20 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7069
15-03-2024 12:48:33 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 7069
15-03-2024 23:07:23 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:07:33 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 302 0
15-03-2024 23:07:34 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
15-03-2024 23:07:34 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
15-03-2024 23:07:35 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
15-03-2024 23:07:35 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
15-03-2024 23:07:41 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'group.position'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 77, in leaderboard
    return render(request, "leaderboard/leaderboard.html", {'users':users_total,
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 137: 'group.position', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
15-03-2024 23:07:41 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 201356
15-03-2024 23:08:04 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 77, in leaderboard
    return render(request, "leaderboard/leaderboard.html", {'users':users_total,
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 146: 'endfor', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
15-03-2024 23:08:04 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 193569
15-03-2024 23:09:09 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 77, in leaderboard
    return render(request, "leaderboard/leaderboard.html", {'users':users_total,
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 146: 'endfor', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
15-03-2024 23:09:09 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 193569
15-03-2024 23:09:28 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 8087
15-03-2024 23:10:29 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 8087
15-03-2024 23:10:35 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 8087
15-03-2024 23:11:26 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:11:27 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:11:34 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 76, in leaderboard
    groups = Group.objects.all().order_by("-group_id")
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\query.py", line 1659, in order_by
    obj.query.add_ordering(*field_names)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\sql\query.py", line 2222, in add_ordering
    self.names_to_path(item.split(LOOKUP_SEP), self.model._meta)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\sql\query.py", line 1725, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'group_id' into field. Choices are: id, name, permissions, user
15-03-2024 23:11:34 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 87232
15-03-2024 23:12:09 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:12:09 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:12:18 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 76, in leaderboard
    groups = Group.objects.all().order_by("-group_id")
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\query.py", line 1659, in order_by
    obj.query.add_ordering(*field_names)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\sql\query.py", line 2222, in add_ordering
    self.names_to_path(item.split(LOOKUP_SEP), self.model._meta)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\sql\query.py", line 1725, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'group_id' into field. Choices are: id, name, permissions, user
15-03-2024 23:12:18 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 87232
15-03-2024 23:12:24 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:12:25 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:12:33 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 83, in leaderboard
    for user_group in journey.user.groups:
TypeError: 'ManyRelatedManager' object is not iterable
15-03-2024 23:12:33 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 72673
15-03-2024 23:13:38 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:13:39 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:13:47 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 85, in leaderboard
    for user_group in user_groups:
TypeError: 'ManyRelatedManager' object is not iterable
15-03-2024 23:13:47 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 72963
15-03-2024 23:14:51 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:14:52 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:14:59 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 84, in leaderboard
    user_groups = journey.user.groups.id
AttributeError: 'ManyRelatedManager' object has no attribute 'id'
15-03-2024 23:14:59 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 72895
15-03-2024 23:15:42 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:15:43 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:15:50 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 85, in leaderboard
    print(user_groups[0])
TypeError: 'ManyRelatedManager' object is not subscriptable
15-03-2024 23:15:50 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 72967
15-03-2024 23:17:07 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:17:07 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:17:26 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:17:26 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:17:35 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:17:36 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:17:47 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:18:39 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:18:39 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:18:52 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:19:25 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:24:49 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:25:52 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:27:51 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
15-03-2024 23:27:55 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3164
15-03-2024 23:28:02 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:28:30 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3164
15-03-2024 23:28:33 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
15-03-2024 23:29:47 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3164
15-03-2024 23:29:51 [38;5;25m[INFO][0m django.server: "GET /groups/search/ HTTP/1.1" 200 2967
15-03-2024 23:29:55 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3164
15-03-2024 23:29:59 [38;5;25m[INFO][0m django.server: "GET /groups/view/4/ HTTP/1.1" 200 2864
15-03-2024 23:30:02 [38;5;25m[INFO][0m django.server: "POST /groups/leave/4/ HTTP/1.1" 302 0
15-03-2024 23:30:04 [38;5;25m[INFO][0m django.server: "GET /groups/view/4/ HTTP/1.1" 200 2869
15-03-2024 23:30:08 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 2945
15-03-2024 23:30:11 [38;5;25m[INFO][0m django.server: "GET /groups/search/ HTTP/1.1" 200 2967
15-03-2024 23:30:18 [38;5;220m[WARNING][0m django.request: Not Found: /groups/group/4/
15-03-2024 23:30:18 [38;5;220m[WARNING][0m django.server: "GET /groups/group/4/ HTTP/1.1" 404 4020
15-03-2024 23:30:22 [38;5;220m[WARNING][0m django.request: Not Found: /groups/group
15-03-2024 23:30:22 [38;5;220m[WARNING][0m django.server: "GET /groups/group HTTP/1.1" 404 4011
15-03-2024 23:30:27 [38;5;220m[WARNING][0m django.request: Not Found: /groups/groups/4/
15-03-2024 23:30:27 [38;5;220m[WARNING][0m django.server: "GET /groups/groups/4/ HTTP/1.1" 404 4023
15-03-2024 23:30:31 [38;5;220m[WARNING][0m django.request: Not Found: /groups/groups/
15-03-2024 23:30:31 [38;5;220m[WARNING][0m django.server: "GET /groups/groups/ HTTP/1.1" 404 4017
15-03-2024 23:30:36 [38;5;220m[WARNING][0m django.request: Not Found: /groups/4/
15-03-2024 23:30:36 [38;5;220m[WARNING][0m django.server: "GET /groups/4/ HTTP/1.1" 404 4002
15-03-2024 23:30:39 [38;5;25m[INFO][0m django.server: "GET /groups HTTP/1.1" 301 0
15-03-2024 23:30:40 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 2945
15-03-2024 23:30:45 [38;5;25m[INFO][0m django.server: "GET /groups/search/ HTTP/1.1" 200 2967
15-03-2024 23:30:50 [38;5;25m[INFO][0m django.server: - Broken pipe from ('127.0.0.1', 63220)
15-03-2024 23:30:51 [38;5;25m[INFO][0m django.server: "GET /groups/search/?q=hype HTTP/1.1" 200 3358
15-03-2024 23:30:54 [38;5;25m[INFO][0m django.server: "GET /groups/view/4/ HTTP/1.1" 200 2869
15-03-2024 23:30:57 [38;5;25m[INFO][0m django.server: "POST /groups/join/4/ HTTP/1.1" 302 0
15-03-2024 23:30:59 [38;5;25m[INFO][0m django.server: "GET /groups/view/4/ HTTP/1.1" 200 2864
15-03-2024 23:31:05 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:31:08 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3164
15-03-2024 23:31:13 [38;5;25m[INFO][0m django.server: "GET /groups/search/ HTTP/1.1" 200 2967
15-03-2024 23:31:20 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9965
15-03-2024 23:31:24 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3164
15-03-2024 23:31:26 [38;5;25m[INFO][0m django.server: "GET /groups/search/ HTTP/1.1" 200 2967
15-03-2024 23:31:30 [38;5;25m[INFO][0m django.server: "GET /groups/search/?q=ccc HTTP/1.1" 200 3766
15-03-2024 23:31:33 [38;5;25m[INFO][0m django.server: "GET /groups/view/3/ HTTP/1.1" 200 2867
15-03-2024 23:31:37 [38;5;25m[INFO][0m django.server: "POST /groups/join/3/ HTTP/1.1" 302 0
15-03-2024 23:31:39 [38;5;25m[INFO][0m django.server: "GET /groups/view/3/ HTTP/1.1" 200 2862
15-03-2024 23:31:42 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3433
15-03-2024 23:34:48 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:34:48 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:34:57 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 83, in leaderboard
    user_groups = journey.user.all()
AttributeError: 'User' object has no attribute 'all'
15-03-2024 23:34:57 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 72489
15-03-2024 23:35:49 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:35:49 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:36:14 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9993
15-03-2024 23:36:37 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9993
15-03-2024 23:37:40 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
15-03-2024 23:38:04 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9993
15-03-2024 23:39:52 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:39:53 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:40:10 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9993
15-03-2024 23:47:38 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:47:38 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:48:00 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9993
15-03-2024 23:49:06 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:49:07 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:49:28 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9993
15-03-2024 23:56:51 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:56:52 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:56:59 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 83, in leaderboard
    user_id = user.id
AttributeError: 'Leaderboard_Entry' object has no attribute 'id'
15-03-2024 23:56:59 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 72887
15-03-2024 23:58:18 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\common\utils.py changed, reloading.
15-03-2024 23:58:18 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:59:04 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:59:04 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
15-03-2024 23:59:21 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 96, in leaderboard
    users.sort(key=lambda x: x.totalCo2Saved, reverse=True)
UnboundLocalError: local variable 'users' referenced before assignment
15-03-2024 23:59:21 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 74074
15-03-2024 23:59:57 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
15-03-2024 23:59:57 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:00:16 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:00:16 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:00:32 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9977
16-03-2024 00:03:46 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:03:46 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:03:57 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 89, in leaderboard
    for group in group_entry:
TypeError: 'Leaderboard_Entry' object is not iterable
16-03-2024 00:03:57 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 73306
16-03-2024 00:04:33 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:04:34 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:04:41 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 8358
16-03-2024 00:04:53 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:04:53 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:05:00 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9977
16-03-2024 00:05:54 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:05:54 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:06:01 [38;5;9m[ERROR][0m django.request: Internal Server Error: /leaderboard/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py", line 92, in leaderboard
    group.totalCo2Saved += user.totalCo2Saved
AttributeError: 'User' object has no attribute 'totalCo2Saved'
16-03-2024 00:06:01 [38;5;9m[ERROR][0m django.server: "GET /leaderboard/ HTTP/1.1" 500 74011
16-03-2024 00:06:30 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:06:30 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:06:37 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 10007
16-03-2024 00:07:04 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
16-03-2024 00:07:08 [38;5;25m[INFO][0m django.server: "GET /groups/ HTTP/1.1" 200 3433
16-03-2024 00:07:15 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 10007
16-03-2024 00:15:30 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:15:30 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:15:38 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 11304
16-03-2024 00:15:38 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9454
16-03-2024 00:16:56 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:16:56 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:17:04 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9977
16-03-2024 00:18:00 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:18:01 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:18:08 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 10005
16-03-2024 00:18:16 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 10005
16-03-2024 00:23:43 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 10005
16-03-2024 00:23:43 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
16-03-2024 00:23:51 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 10005
16-03-2024 00:25:01 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9960
16-03-2024 00:25:09 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9960
16-03-2024 00:25:47 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9986
16-03-2024 00:26:43 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9986
16-03-2024 00:26:48 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 200 4302
16-03-2024 00:26:55 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9986
16-03-2024 00:27:37 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\leaderboard\views.py changed, reloading.
16-03-2024 00:27:38 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 00:27:47 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 9986
16-03-2024 13:57:01 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
16-03-2024 13:57:30 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
16-03-2024 13:57:30 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 9454
16-03-2024 13:57:31 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/forum.jpg HTTP/1.1" 200 627536
16-03-2024 13:57:31 [38;5;220m[WARNING][0m django.request: Not Found: /favicon.ico
16-03-2024 13:57:31 [38;5;220m[WARNING][0m django.server: "GET /favicon.ico HTTP/1.1" 404 3149
16-03-2024 13:57:34 [38;5;25m[INFO][0m django.server: "GET /login/ HTTP/1.1" 200 4071
16-03-2024 13:57:45 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 8864
16-03-2024 14:02:06 [38;5;25m[INFO][0m django.server: "GET /leaderboard/ HTTP/1.1" 200 8860
17-03-2024 22:12:09 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:12:19 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
17-03-2024 22:12:19 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 200 8869
17-03-2024 22:12:22 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4602
17-03-2024 22:12:54 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4608
17-03-2024 22:12:59 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4608
17-03-2024 22:13:07 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4608
17-03-2024 22:13:11 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4608
17-03-2024 22:13:13 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4608
17-03-2024 22:13:14 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4608
17-03-2024 22:14:00 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
17-03-2024 22:14:01 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4608
17-03-2024 22:15:06 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4666
17-03-2024 22:15:07 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4666
17-03-2024 22:15:08 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4666
17-03-2024 22:15:09 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4666
17-03-2024 22:15:27 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4666
17-03-2024 22:15:38 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4660
17-03-2024 22:15:41 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4660
17-03-2024 22:16:57 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4650
17-03-2024 22:17:50 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4611
17-03-2024 22:27:08 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4611
17-03-2024 22:27:10 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4611
17-03-2024 22:27:11 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4611
17-03-2024 22:27:12 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4611
17-03-2024 22:27:42 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4615
17-03-2024 22:27:44 [38;5;220m[WARNING][0m django.request: Not Found: /PrivacyPolicy
17-03-2024 22:27:44 [38;5;220m[WARNING][0m django.server: "GET /PrivacyPolicy HTTP/1.1" 404 3155
17-03-2024 22:28:08 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\register\views.py changed, reloading.
17-03-2024 22:28:09 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:28:15 [38;5;220m[WARNING][0m django.request: Not Found: /PrivacyPolicy
17-03-2024 22:28:15 [38;5;220m[WARNING][0m django.server: "GET /PrivacyPolicy HTTP/1.1" 404 3155
17-03-2024 22:28:18 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4624
17-03-2024 22:28:19 [38;5;220m[WARNING][0m django.request: Not Found: /register/PrivacyPolicy
17-03-2024 22:28:19 [38;5;220m[WARNING][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 404 3246
17-03-2024 22:29:41 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\register\views.py changed, reloading.
17-03-2024 22:29:41 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:29:56 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4624
17-03-2024 22:29:58 [38;5;220m[WARNING][0m django.request: Not Found: /register/PrivacyPolicy
17-03-2024 22:29:58 [38;5;220m[WARNING][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 404 3246
17-03-2024 22:34:54 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\carbonCommuter\urls.py changed, reloading.
17-03-2024 22:34:54 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:37:05 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:37:17 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\carbonCommuter\urls.py changed, reloading.
17-03-2024 22:37:18 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:37:36 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\carbonCommuter\urls.py changed, reloading.
17-03-2024 22:37:36 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:37:42 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
17-03-2024 22:37:42 [38;5;25m[INFO][0m django.server: "GET /static/carbonCommuter/style.css HTTP/1.1" 304 0
17-03-2024 22:37:50 [38;5;25m[INFO][0m django.server: "GET /login/ HTTP/1.1" 200 4071
17-03-2024 22:37:55 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 302 0
17-03-2024 22:37:55 [38;5;25m[INFO][0m django.server: "GET /accounts/login/?next=/user/ HTTP/1.1" 200 4071
17-03-2024 22:37:59 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 302 0
17-03-2024 22:37:59 [38;5;25m[INFO][0m django.server: "GET /accounts/login/?next=/user/ HTTP/1.1" 200 4071
17-03-2024 22:41:57 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4620
17-03-2024 22:41:59 [38;5;220m[WARNING][0m django.request: Not Found: /register/views.PrivacyPolicy
17-03-2024 22:41:59 [38;5;220m[WARNING][0m django.server: "GET /register/views.PrivacyPolicy HTTP/1.1" 404 3264
17-03-2024 22:42:19 [38;5;220m[WARNING][0m django.request: Not Found: /register/views.PrivacyPolicy
17-03-2024 22:42:19 [38;5;220m[WARNING][0m django.server: "GET /register/views.PrivacyPolicy HTTP/1.1" 404 3264
17-03-2024 22:42:21 [38;5;220m[WARNING][0m django.request: Not Found: /register/views.PrivacyPolicy
17-03-2024 22:42:21 [38;5;220m[WARNING][0m django.server: "GET /register/views.PrivacyPolicy HTTP/1.1" 404 3264
17-03-2024 22:42:27 [38;5;220m[WARNING][0m django.request: Not Found: /register/PrivacyPolicy
17-03-2024 22:42:27 [38;5;220m[WARNING][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 404 3246
17-03-2024 22:45:00 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\carbonCommuter\urls.py changed, reloading.
17-03-2024 22:45:01 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:46:56 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\register\views.py changed, reloading.
17-03-2024 22:46:57 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:48:05 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\carbonCommuter\urls.py changed, reloading.
17-03-2024 22:48:06 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:48:47 [38;5;25m[INFO][0m django.server: "GET / HTTP/1.1" 200 2961
17-03-2024 22:48:52 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4616
17-03-2024 22:48:54 [38;5;220m[WARNING][0m django.request: Not Found: /PrivacyPolicy/
17-03-2024 22:48:54 [38;5;220m[WARNING][0m django.server: "GET /PrivacyPolicy/ HTTP/1.1" 404 3158
17-03-2024 22:49:13 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4614
17-03-2024 22:49:16 [38;5;220m[WARNING][0m django.request: Not Found: /register/PrivacyPolicy
17-03-2024 22:49:16 [38;5;220m[WARNING][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 404 3246
17-03-2024 22:49:31 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\register\urls.py changed, reloading.
17-03-2024 22:49:31 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:49:33 [38;5;220m[WARNING][0m django.request: Not Found: /register/PrivacyPolicy
17-03-2024 22:49:33 [38;5;220m[WARNING][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 404 3439
17-03-2024 22:49:34 [38;5;220m[WARNING][0m django.request: Not Found: /register/PrivacyPolicy
17-03-2024 22:49:34 [38;5;220m[WARNING][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 404 3439
17-03-2024 22:49:41 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\register\urls.py changed, reloading.
17-03-2024 22:49:42 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:49:44 [38;5;25m[INFO][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 200 4607
17-03-2024 22:51:27 [38;5;25m[INFO][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 200 4651
17-03-2024 22:51:31 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4614
17-03-2024 22:52:04 [38;5;25m[INFO][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 200 4651
17-03-2024 22:52:06 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4614
17-03-2024 22:52:10 [38;5;25m[INFO][0m django.server: "GET /register/PrivacyPolicy HTTP/1.1" 200 4651
17-03-2024 22:52:12 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4614
17-03-2024 22:54:46 [38;5;25m[INFO][0m django.utils.autoreload: C:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\register\views.py changed, reloading.
17-03-2024 22:54:47 [38;5;25m[INFO][0m django.utils.autoreload: Watching for file changes with StatReloader
17-03-2024 22:54:50 [38;5;25m[INFO][0m django.server: "GET /register/ HTTP/1.1" 200 4614
17-03-2024 22:55:08 [38;5;25m[INFO][0m django.server: "POST /register/ HTTP/1.1" 200 4715
17-03-2024 22:55:32 [38;5;9m[ERROR][0m django.request: Internal Server Error: /register/
Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\mysql\base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\MySQLdb\cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\MySQLdb\cursors.py", line 330, in _query
    db.query(q)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\MySQLdb\connections.py", line 261, in query
    _mysql.connection.query(self, query)
MySQLdb.IntegrityError: (1364, "Field 'streak' doesn't have a default value")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "c:\Users\downw\OneDrive\Documents\ECM2434_Group_Software_Engineering_Project\carbonCommuter\register\views.py", line 74, in register
    profile.save()
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\base.py", line 1020, in _save_table
    results = self._do_insert(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\base.py", line 1061, in _do_insert
    return manager._insert(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\models\sql\compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\db\backends\mysql\base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\MySQLdb\cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\MySQLdb\cursors.py", line 330, in _query
    db.query(q)
  File "C:\Users\downw\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\MySQLdb\connections.py", line 261, in query
    _mysql.connection.query(self, query)
django.db.utils.IntegrityError: (1364, "Field 'streak' doesn't have a default value")
17-03-2024 22:55:32 [38;5;9m[ERROR][0m django.server: "POST /register/ HTTP/1.1" 500 176582
17-03-2024 22:56:03 [38;5;25m[INFO][0m django.server: "POST /register/ HTTP/1.1" 200 4694
17-03-2024 22:56:08 [38;5;25m[INFO][0m django.server: "GET /login/ HTTP/1.1" 200 4071
17-03-2024 22:56:11 [38;5;25m[INFO][0m django.server: "GET /user/ HTTP/1.1" 302 0
17-03-2024 22:56:11 [38;5;25m[INFO][0m django.server: "GET /accounts/login/?next=/user/ HTTP/1.1" 200 4071
17-03-2024 22:56:13 [38;5;25m[INFO][0m django.server: - Broken pipe from ('127.0.0.1', 57950)
